private/scripts/sage-theme-install.sh

```diff
@@ -144,23 +143,23 @@ function check_login() {
 get_field() {
   local input="$1"
   # Remove leading and trailing whitespace from each line
-  input="$(echo "$input" | sed -e 's/^[ \t]*//')"
+  input="${input#"${input%%[![:space:]]*}"}"
 
   # Remove the first and last lines that are entirely dashes
   input="$(echo "$input" | sed -e '1d' -e '$d')"
   # $1: field name
   # $2: input string
-  echo "$2" | awk -v field="$1" '$1 == field { print $2 }'
+  echo "$2" | awk -v field="$1" '$1 -eq field { print $2 }'
```

Comment:
Sorry for the confusion with my above comment. -eq is only for numeric comparisons. I think you want == here.
```suggestion
  echo "$2" | awk -v field="$1" '$1 == field { print $2 }'
```